utils/open-vm-tools/Makefile

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=open-vm-tools
+PKG_VERSION:=10.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/vmware/open-vm-tools.git
+PKG_SOURCE_VERSION:=stable-$(PKG_VERSION)
+
+PKG_FIXUP:=autoreconf
+PKG_LICENSE:=LGPL-2.1
```

Comment:
`PKG_LICENSE_FILES := open-vm-tools/LICENSE` can be added here.

Reply:
Fixed.